velox/docs/functions/spark/coverage.rst

```diff
@@ -0,0 +1,169 @@
+=================
+Function Coverage
+=================
+
+Here is a list of all scalar and aggregate scala functions with functions that are available in Velox highlighted.
```

Comment:
Would you update PR description to explain how you generated this file? Perhaps, add a README with instructions how to update this file in the future.

Reply:
Sorry, I have only now discovered that there is a program called velox_prestosql_coverage.  I used to write directly by hand. So I think I should make another one called velox_spark_coverage for spark function.